io.httpie.Httpie.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Name=HTTPie
+Comment=HTTPie for Desktop
+Exec=httpie %U
+Terminal=false
+Type=Application
+Icon=io.httpie.Httpie
+StartupWMClass=HTTPie
+MimeType=x-scheme-handler/pie;x-scheme-handler/pie;
```

Comment:
duplicate?

Reply:
Oops, yes, sorry, that duplicate is in the upstream .desktop file, I forgot to remove it